mesh/accountsettings/blockedaccountbridge.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS BlockedAccountBridge(
+	settingID INT,
```

Comment:
1. I think this can be `blockerAccountID` if we decide to remove `settingID` from `AccountSettings` (*refer to my other comment about the necessity of `settingID`*). Examples of this relationship:
    1. Bob (1) has blocked Chris (2) 
        1. `blockerAccountID = 1`
        2. `blockedAccountID = 2`
    1. Bob (1) has blocked Tony (3)
        1. `blockerAccountID = 1`
        2. `blockedAccountID = 3`  
1. nitpick: there's an extra indentation here